test/StreamHelper.php

```diff
@@ -27,7 +27,9 @@ protected function createTemporaryResource($content = null)
     public static function tearDownAfterClass(): void
     {
         foreach (static::$tempFiles as $tempFile) {
-            @unlink($tempFile);
+            if(file_exists($tempFile)){
```

Comment:
Two things:

1. This is not PSR-2 formatting (missing space in `if (`)
2. This should use `is_file()` instead of `file_exists` to ensure that directories are not targeted